.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 
 .idea/
+
+.cursorrules
```

Comment:
@theborakompanioni this .cursorrules is for anyone using the Cursor editor, it is trending on X and I also use it, hence added this for future contributors.

Reply:
Does not hurt to keep it! :pray: